src/microbe_stage/SpawnSystem.cs

```diff
@@ -293,8 +293,11 @@ private void SpawnEntities(Vector3 playerPosition, Vector3 playerRotation, int e
 
         var spawner = enumerable.GetEnumerator();
 
+        var any = false;
```

Comment:
Isn't this duplicate logic for what `spawnsLeftThisFrame` does?

Reply:
using `spawnsLeftThisFrame` I can't differenciate if spawner contains any elements.

Reply:
Then change the logic so that it can be used for that.
If you set max spawns to 1 it should do the same thing.
![Kuvakaappaus - 2020-12-27 15-12-51](https://user-images.githubusercontent.com/3796411/103171575-0567ad00-4856-11eb-8114-4a42e49bfed0.png)